src/test/kotlin/no/nav/syfo/oppfolgingstilfelle/kafka/KafkaOppfolgingstilfellePersonServiceSpek.kt

```diff
@@ -573,44 +560,6 @@ class KafkaOppfolgingstilfellePersonServiceSpek : Spek({
                     }
                 }
             }
-            describe("Aktivitetskrav(AVVENT) exists for oppfolgingstilfelle, then oppfolgingstilfelle gradert") {
```

Comment:
Noen tester her som kunne fjernes siden vi ikke lenger har noe logikk for om aktivitetskravet er vurdert -> så blir gradert -> så ikke gradert.